fbpic/particles/utilities/utility_methods.py

```diff
@@ -102,54 +102,3 @@ def weights(x, invdx, offset, Nx, direction, shape_order):
 
     # Return the result
     return( i, S )
```

Comment:
This has been moved into the file that contains the `ContinuousInjector` object.